releasing/plugin-commands-publishing/src/publish.ts

```diff
@@ -233,8 +246,24 @@ Do you want to continue?`,
     packDestination,
   })
   await copyNpmrc({ dir, workspaceDir: opts.workspaceDir, packDestination })
+
+  const { publishConfig } = manifest
+  const registry = publishConfig?.registry ?? opts.registries.default
```

Comment:
What is the expected behaviour when there is a custom registry for the package's scope? I assume the registry for the scope should be used.

Reply:
I thought it would always use the `publishConfig` from the `package.json`, after a little research looks like it works in npm with the `.npmrc` too. I'll fix it.

Reply:
@zkochan I added a commit that just sends the tokens and lets npm decide which registry has to be used. Is it ok?

Reply:
Also I added support for argv helper tokens (for testing support) and the `.js` helpers will use node in Windows.

Reply:
ok, that should work